modules/orca/main.tf

```diff
@@ -122,11 +130,11 @@ module "orca_iam" {
   ## Cumulus Variables
   ## --------------------------
   ## REQUIRED
-  buckets                  = var.buckets
+  buckets                  = module.orca_s3.buckets
```

Comment:
I think this should also be updated 
https://github.com/nasa/cumulus-orca/blob/4d6cff91690984c4a8b947d89a2115d1e0009301/main.tf#L17

maybe verify with `terraform validate`